specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -10744,6 +10972,160 @@
       },
       "type": "object"
     },
+    "AntispamMailDirection": {
+      "description": "Email Directionality Status.",
+      "enum": [
+        "Unknown",
+        "Inbound",
+        "Outbound",
+        "Intraorg"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "DeliveryAction",
+        "values": [
+          {
+            "description": "Unknown direction status",
+            "value": "Unknown"
+          },
+          {
+            "description": "Inbound direction status",
+            "value": "Inbound"
+          },
+          {
+            "description": "Outbound direction status",
+            "value": "Outbound"
+          },
+          {
+            "description": "Internal direction status",
+            "value": "Intraorg"
+          }
+        ]
+      }
+    },
+    "DeliveryAction": {
+      "description": "The email delivery action.",
+      "enum": [
+        "Unknown",
+        "DeliveredAsSpam",
+        "Delivered",
+        "Blocked",
+        "Replaced"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "modelAsString": true,
+        "name": "DeliveryAction",
```

Comment:
We have two enum definitions with the same name, but different value set (one of them has 4 entries, and the other has 5). Could you please resolve this?

Reply:
Thanks! copy&paste mistake...
Fixed.